src/ui/FieldValue/FieldValue.ts

```diff
@@ -270,6 +271,7 @@ export class FieldValue extends Component {
       }
     }
 
+    toRender = StringUtils.capitalizeFirstLetter(toRender);
```

Comment:
Heh no.

This will capitalize all values output by that component.

It's not used only by date.